hack/apply-base-cluster-resources

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+declare -r SCRIPT_DIR=$(cd $(dirname ${0}) >/dev/null 2>&1 && pwd)
+declare -r REPO_ROOT=$(cd $(dirname ${SCRIPT_DIR}) >/dev/null 2>&1 && pwd)
+
+declare -r HELM_OPERATOR_VERSION=v1.0.0-rc8
+declare -r SECRETS_MANAGER_VERSION=release-1.0.2
+
+declare -ra CRDS=(
+  https://raw.githubusercontent.com/fluxcd/helm-operator/${HELM_OPERATOR_VERSION}/deploy/flux-helm-release-crd.yaml
+  https://raw.githubusercontent.com/tuenti/secrets-manager/${SECRETS_MANAGER_VERSION}/config/crd/bases/secrets-manager.tuenti.io_secretdefinitions.yaml
+)
```

Comment:
After the full cluster rebuild, I tried re-applying all the k8s / Helm stuff. The Helm releases deadlocked because Flux enqueued the secret-manager release after a bunch of releases that depended on secrets being present, and it wouldn't progress through the queue until those blocked releases reached a terminal state.

Instead of doing "all k8s resources" in one script and "all Helm releases" in another, I shifted to "all k8s + Helm releases required to make any cluster work" (Flux and secrets-manager) and "k8s + Helm releases for the command-center"

Reply:
I take it there's no way to get flux to do things in any particular order?

Reply:
Not that I could find.

eVeNtUaL cOnSiStEnCy